client/src/clientRoutes/reviews.js

```diff
@@ -20,5 +20,38 @@ module.exports = {
     return axios.put(
       `/reviews/${review_id}/report`
     );
+  },
+  reviewAdd: (reviewData, product_id) => {
+    let newChars = {};
+    let oldChars = Object.assign(reviewData.characteristics);
+
+    for (let char in oldChars) {
+      newChars[char]= parseInt(oldChars[char]);
+    }
+
+    let review = {
+      product_id: product_id,
+      rating: parseInt(reviewData.rating),
+      summary: reviewData.summary,
+      body: reviewData.body,
+      recommend: reviewData.recommend === 'No' ? false : true,
+      name: reviewData.name,
+      email: reviewData.email,
+      photos: reviewData.photos,
+      characteristics: newChars
+    }
+
+    console.log(review);
+    return axios.post(`/reviews/add`, review)
+  },
+  reviewsInteraction: (event) => {
+    let interaction = event.target.getAttribute('interaction');
+    axios.post(
+      `/reviews/interaction`, {
+        element: interaction,
+        widget: 'Ratings And Reviews - Team Cotija',
+        time: new Date()
```

Comment:
is this actually giving you the correct date?  for me i had to use

`new Data(Date.now(currentTimestamp))`

to get the current date.  Without it, it was populating a date of like dec 31 1969 lol (the date comps went live)